dist/stat.py

```diff
@@ -90,6 +92,15 @@ def print_defines():
  */
 ''')
     print_defines_one('JOIN', 3000, join_stats)
+    f.write('''
+/*!
+ * @}
+ * @name Statistics for session operation
+ * @anchor statistics_session_op
+ * @{
+ */
+''')
+    print_defines_one('SESN', 4000, session_op_stats)
```

Comment:
To address the comment https://github.com/wiredtiger/wiredtiger/pull/4359#discussion_r233245628. This would give us the macro name like this:
`#define WT_STAT_SESN_OP_BYTES_READ                      4000`
I looks readable. Let me know if you would rather prefer `WT_STAT_SESSION_OP_BYTES_READ`.